packages/gatsby-theme-newrelic/README.md

```diff
@@ -603,6 +606,28 @@ import { Feedback } from '@newrelic/gatsby-theme-newrelic';
 />
 ```
 
+### `GlobalFooter`
+
+Renders the global footer used on all New Relic Gatsby sites. This component utilizes the [`layout` configuration](#layout) from the theme to size itself and the `siteMetadata` for the repository URL.
+
+_NOTE_: The logo displayed in the footer is a generic to New Relic logo, but can be changed with [shadowing](https://www.gatsbyjs.com/docs/themes/shadowing).
+
+```js
+import { GlobalFooter } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Props**
+
+| Prop               | Type    | Required | Default | Description                                                                                                   |
```

Comment:
Add documentation for the `className` support.

Reply:
Done in https://github.com/newrelic/gatsby-theme-newrelic/pull/134/commits/a0541c34052b61fead621e6f1539ef3c9e60a5b1